android/app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="default_margin">24dp</dimen>
```

Comment:
I think we will forget what is the `default_margin` value in the future soon 😄  Should we add the value at the end of the resource's name? `default_margin_24`

Reply:
Can. But is it really nice to have?

Reply:
Yep. I am still fine with the current one 😄  But just make sure we have to keep in mind `default` = 24. You can leave it as it is for now 😄